src/frontend/src/pages/WorkPackageDetailPage/WorkPackageViewContainer/WorkPackageDetails.tsx

```diff
@@ -3,64 +3,107 @@
  * See the LICENSE file in the repository root folder for details.
  */
 
-import { WorkPackage } from 'shared';
+import { WorkPackage, wbsPipe } from 'shared';
 import { percentPipe, fullNamePipe, datePipe, weeksPipe } from '../../../utils/pipes';
 import WbsStatus from '../../../components/WbsStatus';
-import PageBlock from '../../../layouts/PageBlock';
-import { Box, Grid } from '@mui/material';
+import { Box, Divider, Grid, Link, Stack, Typography } from '@mui/material';
 import DetailDisplay from '../../../components/DetailDisplay';
 import WorkPackageStageChip from '../../../components/WorkPackageStageChip';
 import { timelinePipe } from '../../../utils/pipes';
+import { Construction, Work } from '@mui/icons-material';
+import ScheduleIcon from '@mui/icons-material/Schedule';
+import StackedLineChartIcon from '@mui/icons-material/StackedLineChart';
+import DoneAllIcon from '@mui/icons-material/DoneAll';
+import { Link as RouterLink } from 'react-router-dom';
+import { routes } from '../../../utils/routes';
 
 interface WorkPackageDetailsProps {
   workPackage: WorkPackage;
+  dependencies: WorkPackage[];
 }
 
-const WorkPackageDetails: React.FC<WorkPackageDetailsProps> = ({ workPackage }) => {
+const WorkPackageDetails: React.FC<WorkPackageDetailsProps> = ({ workPackage, dependencies }) => {
   return (
-    <PageBlock
-      title={'Work Package Details'}
-      headerRight={
-        <Box>
-          {workPackage.stage ? <WorkPackageStageChip stage={workPackage.stage} /> : null}
-          <WbsStatus status={workPackage.status} />
-        </Box>
-      }
-    >
-      <Grid container spacing={1}>
-        <Grid item xs={4} md={4}>
-          <DetailDisplay label="Lead" content={fullNamePipe(workPackage.projectLead)} paddingRight={2} />
+    <>
+      <Grid container columnSpacing={2} justifyContent="flex-start" sx={{ mb: 2, mt: 1 }}>
+        <Grid item display="flex" alignItems="center">
+          <Typography variant="h5">Work Package Details</Typography>
         </Grid>
+        <Grid item display="flex" alignItems="center">
```

Comment:
again this is not flexing anyting

Reply:
ah I see what these are for. we should just delete all the grid items and then make the big grid a box and then make it flex and then align the items as center, just tried it and it looks the same